index.html

```diff
@@ -19,23 +19,87 @@
         <nav class="nav-bar">
             <img id="shapes" src="./images/shapes.svg" alt="">
             <img id="hamburger" src="./images/ic_menu.svg" alt="">
+            <ul class="menu">
+                <li><a href="">Hello</a></li>
+                <li><a href="">Portfolio</a></li>
+                <li><a href="">About</a></li>
+                <li><a href="">Contact</a></li>
+            </ul>
         </nav>
     </header>
-    
     <section class="headline">
-        <h1 class="title">Emmanuel Cortés</h1>
-        <p class="presentation">Hello! I am a software developer! 
+        <div class="hero">
+          <h1 class="title">Emmanuel Cortés</h1>
```

Comment:
- [ ] If you can make your title in two lines like in the Figma template, that would be great. 🤗
#### your design 👇️
![Screenshot from 2022-11-03 20-50-20](https://user-images.githubusercontent.com/89228743/199831429-1c615bd2-436b-42f1-941e-480195e00a36.png)
### The Figma design 👇️
![Screenshot from 2022-11-03 20-52-25](https://user-images.githubusercontent.com/89228743/199831721-f6e8b5c9-59df-4eb5-9397-69d6478bd44c.png)